shacl12-sparql/index.html

```diff
@@ -2261,8 +2261,9 @@ <h2>Security and Privacy Considerations</h2>
 		<section id="ack" class="appendix informative">
 			<h2>Acknowledgements</h2>
 			<p>
-				The original 1.0 version of SHACL was produced by the RDF Data Shapes Working Group.
-				See its <a href="https://www.w3.org/TR/shacl/#ack">SHACL 1.0 Acknowledgements section</a>.
+				Original SHACL core specifications were produced by the RDF Data Shapes Working Group.
+				See the <a href="https://www.w3.org/TR/shacl/#ack">Core specification's Acknowledgements section</a> and
+        the <a href="https://www.w3.org/TR/shacl-af/#ack">Advanced Features specification's Acknowledgements section</a>.
```

Comment:
Because these are back-in-time links, I think they should use the dated URLs.

```suggestion
				See the <a href="https://www.w3.org/TR/2017/REC-shacl-20170720/#ack">Core specification's Acknowledgements section</a> and
        the <a href="https://www.w3.org/TR/2017/NOTE-shacl-af-20170608/#ack">Advanced Features specification's Acknowledgements section</a>.
```